esquery.js

```diff
@@ -54,64 +52,64 @@ function matches(node, selector, ancestry) {
         case 'identifier':
             return selector.value.toLowerCase() === node.type.toLowerCase();
 
-        case 'field':
-            path = selector.name.split('.');
-            ancestor = ancestry[path.length - 1];
+        case 'field': {
+            const path = selector.name.split('.');
+            const ancestor = ancestry[path.length - 1];
             return inPath(node, ancestor, path);
 
-        case 'matches':
-            for (i = 0, l = selector.selectors.length; i < l; ++i) {
+        } case 'matches':
```

Comment:
Blocks on their own line, please.

Reply:
I've added for the new `case`'s. Let me know if you want to apply https://eslint.org/docs/rules/brace-style with the `"stroustrup"` option to enforce this for all block types (I don't think there are rules for doing so only for cases).

Reply:
Oh no, `stroustrup` is the worst. `1tbs` seems fine and should support this usage in the `case` body.

Reply:
It turns out `brace-style` doesn't actually enforce on `case`, but I've added it anyways as it was consistent with current usage (when using `1tbs` and the `allowSingleLine` option).